### PR TITLE
Input: fix 密码输入框无值时点击'切换密码图标'不能切换

### PR DESCRIPTION
**Pull request by yi-huan**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.
使用了 `show-password` 的密码输入框当值为空时，点击“切换密码图标”，输入框类型不能切换。

### CODE REVIEW

**Comment by iamkun** [comment]
请解决一下冲突 rebase dev

File: packages/input/src/input.vue [hunk 1]
```diff
@@ -64,7 +64,7 @@
           ></i>
           <i v-if="showPwdVisible"
             class="el-input__icon el-icon-view el-input__clear"
-            @click="handlePasswordVisible"
+            @mousedown="handlePasswordVisible"
```
Comment (luckyCao) [inline packages/input/src/input.vue]: 为啥改成 mousedown

Comment (yi-huan) [reply packages/input/src/input.vue]: 目前输入框为空的时候，点击切换密码按钮，不能切换。
改成 `mousedown`，优先级高于 `input` 的 `focus` 事件

**Comment by luckyCao** [comment]
Sorry, we have no plan to support this.

**Pull request closed without merging**